src/mgr/MgrCommands.h

```diff
@@ -100,10 +100,13 @@ COMMAND("osd test-reweight-by-pg " \
 
 COMMAND("osd scrub " \
 	"name=who,type=CephString", \
-	"initiate scrub on osd <who>", "osd", "rw", "cli,rest")
+	"initiate scrub on osd <who>, or use <all|any|*> to scrub all", \
```

Comment:
as per discussion with sage at https://github.com/ceph/ceph/pull/15381

`It would be nice to move this into a helper, though, so that the other commands that take lists of osds can also inherit the 'any' syntax (and maybe we can also make 'all' or '*' work too?).`

> could you point me any of the places where we are following it?

`osd down`、`osd out`、etc.